docker-compose.ci.yml

```diff
@@ -0,0 +1,4 @@
+version: '3'
+
+# to add images that are only used for tests
+services: {}
```

Comment:
Should we add ci images here?

Reply:
Jenkins relies on this file existing in the build phase, but in this version of the Jenkinsfile there is no need to put anything into it